modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/CalculateWGACoverage.pm

```diff
@@ -184,16 +189,24 @@ sub run {
 						  intron_length            => $combined_coverage->{intron_len},
 						}
 					);
+					$sum_scores += $combined_coverage->{score};
+					$n_scores ++;
 				}
                 # disconnect from alignment_db
                 $dba->dbc->disconnect_if_idle();
+
+				my $avg_score = $sum_scores / $n_scores; # $n_scores should be 2
```

Comment:
If `$n_scores` should be 2, accumulated as it is on line 193, perhaps a handy warning to flag when it is not 2?

Reply:
:+1: I have added a commit: https://github.com/Ensembl/ensembl-compara/commit/fdf464f43ed9202e4da9e155842fe9744fb0b8fc